codejam3-pokemon/Program.cs

```diff
@@ -28,80 +31,80 @@ public static async Task Main(string[] args)
                                 `'                            '-._|                                                                                       
                                                                            ";
 
-        Console.WriteLine(title);
-
-        Console.WriteLine("Welcome to the Pokedex!");
-
-        //Console.WriteLine("Enter the name of a Pokemon to see its stats!");
-
-        //var p = Console.ReadLine();
-
-        bool quit = false;
+            Console.WriteLine(title);
+            Console.WriteLine("Welcome to the Pokedex!");
+        await startGame();
+    }
 
-        while (!quit)
+    public static async Task startGame()
```

Comment:
modularizing the program into functions makes it more readable when the functions are called and it makes it easier to move things around.